.isort.cfg

```diff
@@ -0,0 +1,2 @@
+[settings]
+known_third_party = aniso8601,graphql,graphql_relay,promise,pytest,pytz,pyutils,setuptools,six,snapshottest,sphinx_graphene_theme
```

Comment:
Might be nice to put these on separate lines for easier readability and less diff noise in the future.